content/docs/user-guide/components/reference/gradients/image-gradient.md

```diff
@@ -21,23 +22,53 @@ Add the **Image Gradient** component to generate a gradient from an image, with
-| **Preview Size** | Sets the dimensions of the preview. | Vector3: 0.0 to Infinity | X:`1.0`, Y:`1.0`, Z:`1.0` |
+| **Preview Size** | Sets the dimensions of the preview.<br> <br>*This field is available only if there is no entity selected in **Pin Preview to Shape**.* | Vector3: 0.0 to Infinity | X:`1.0`, Y:`1.0`, Z:`1.0` |
 | **Constrain to Shape** | If `Enabled`, the gradient preview uses the bounds of the entity selected in **Pin Preview to Shape**.<br> <br>*This field is available only if an entity is selected in **Pin Preview to Shape**.* | Boolean | `Disabled` |
-| **Image Asset** | Sets the source image to generate this gradient's values. | Gradient Image Asset | None |
-| **Tiling X** | Sets the number of times to tile the image horizontally. | Float: 0.01 to Infinity | `1.0` |
-| **Tiling Y** | Sets the number of times to tile the image vertically. | Float: 0.01 to Infinity | `1.0` |
+| **Tiling** | Sets the number of times to tile the image horizontally (X) and vertically (Y). | Vector2: 0.01 to Infinity | X: `1.0`, Y: `1.0` |
+| **Image Asset** | Sets the source **AZ::RPI::StreamingImageAsset** to generate this gradient's values. | **AZ::RPI::StreamingImageAsset** | None |
+
+### Advanced properties
+
+The **Advanced** group of properties has an in-line toggle switch that enables/disables the entire group of advanced properties. By default, these are disabled.
+
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **Channel To Use** | The channel component of the image to sample from.<br><br>The `Terrarium` option is for an image-based terrain file format as defined here: https://www.mapzen.com/blog/terrain-tile-service/ | `Red`, `Green`, `Blue`, `Alpha`, `Terrarium` | `Red` |
+| **Custom Scale** | Choose a scaling operation to be applied to all image data.<br><br>The `Auto` option will automatically scale based on the min/max values in the image data. | `None`, `Auto`, `Manual` | `None` |
+| **Range Minimum** | The minimum value used to scale image data into the `0`-`1` range for the gradient. All image values at or below the minimum will be scaled to `0`.<br> <br>*This field is available only if the **Custom Scale** field is set to `Manual`.* | Float: `0.0` to `1.0`  | `0.0` |
+| **Range Maximum** | The maximum value used to scale image data into the `0`-`1` range for the gradient. All image values at or above the maximum will be scaled to `1`.<br> <br>*This field is available only if the **Custom Scale** field is set to `Manual`.* | Float: `0.0` to `1.0`  | `1.0` |
+| **Mip Index** | Specify which mip level index to sample from.<br><br>If you specify a mip level higher than the number of mip levels available in your image, then the highest existing mip index will be used. | Integer: `0` to `AZ::RHI::Limits::Image::MipCountMax` (15) | `0` |
```

Comment:
I'm wondering if we need to call out MipCountMax here, will a user need to know this when using this component regularly? 

If you want to keep it in, then I suggest moving the MipCountMax to the description (keeps the values easy to scan). Also, you can just minimize to "MipCountMax" or even "Image::MipCountMax" and link to it on GitHub. Makes it easy to find!

```suggestion
| **Mip Index** | Specify which mip level index to sample from.<br><br>If you specify a mip level higher than the number of mip levels available in your image, then the highest existing mip index will be used (see [`MipCountMax`](https://github.com/o3de/o3de/blob/development/Gems/Atom/RHI/Code/Include/Atom/RHI.Reflect/Limits.h)). | Int: `0` to `15` | `0` |
```

Reply:
Yeah the user wouldn't really need to know `MipCountMax` by name, I just wanted to give some context for why it would be `15`. I like your idea for linking to it so I will do that!